MyStore/MyStore.cpp

```diff
@@ -1,12 +1,17 @@
 ﻿#include "JsonParcer.h"
+#include "GeneratorData.h"
 
 
 int main(int argc, char *argv[])
 {
-
-	JsonParcer obj("..\\MyStore\\data\\Data.json");
-	obj.Deserialization();
+	while (true)
+	{
+			char c = _getche();
+			if (c == ' ')
+				break;
+	} 

```

Comment:
Formatting is incorrect, run clang format (Ctrl+K, Ctrl+F) on all files to fix formatting issues.